README.md

```diff
@@ -49,17 +49,16 @@ def test_request_to_mock_server(mock_server):
 
     with mock_server(imposter):
         # Make request to mock server - exercise code under test here
-        response = requests.get("{}/test".format(imposter.url))
+        response = requests.get(f"{imposter.url}/test")
 
         assert_that("We got the expected response", 
                     response, is_response().with_status_code(200).and_body("sausages"))
         assert_that("The mock server recorded the request", 
                     imposter, had_request().with_path("/test").and_method("GET"))
 ```
+Imposter will be killed after `with`.
 
-Needs a [pytest fixture](https://docs.pytest.org/en/latest/fixture.html), most easily defined in [`conftest.py`]
-(https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions):
-
+If you need a [pytest fixture](https://docs.pytest.org/en/latest/fixture.html), most easily it can be defined in [`conftest.py`](https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions):
```

Comment:
The example shown does indeed require a `conftest.py` so let's put this back?

Reply:
Ok, reverted